realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -279,29 +281,36 @@ public RealmAsyncTask makeOffer(PermissionOffer offer, final Callback<String> ca
     }
 
     /**
-     * FIXME
-     * @param offerToken
-     * @param callback
-     * @return
+     * Accepts a permission offer sent by another user. Once this offer is accepted successfully, the permissions
+     * described by the token will be granted.
```

Comment:
Yes. The offer/accept API is intended for async use cases like email/chat/app-links etc.